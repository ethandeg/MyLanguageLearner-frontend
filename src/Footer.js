
const Footer = () => {
    return (
        <footer className="footer" >
            <div className="container">
                <div className="columns">
                    <div className="column is-half">
                        <div className="content is-medium">
                            <h6>My Language Learner</h6>
                            <hr></hr>
                            <p>Designed, Coded, and Created by Ethan Degenhardt</p>
                            <p>ethandeg1996@gmail.com</p>
                            <p>(636) 373-3435</p>
                        </div>

                    </div>

                    <div className="column is-half">
                        <div className="content is-medium">
                            <h6>Creation</h6>
                            <hr />
                            <p className="">This app was made with the following technology stack:</p>
                            <ul className="content is-small">
                                <li>NodeJS</li>
                                <li>Express</li>
                                <li>React</li>
                                <li>Redux</li>
                                <li>Bulma CSS</li>
                            </ul>
                        </div>

                    </div>
                </div>
            </div>
        </footer>
    )
}


export default Footer