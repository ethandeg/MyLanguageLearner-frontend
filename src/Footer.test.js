import { render, screen } from '@testing-library/react';
import Footer from './Footer';
//smoke test
test('renders without crashing', () => {
  render(<Footer />);

});

//snapshot test

test("matches snapshot", () => {
    const {asFragment} = render(<Footer />)
    expect (asFragment()).toMatchSnapshot()
})
