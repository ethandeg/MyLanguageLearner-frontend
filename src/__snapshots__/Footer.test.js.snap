// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <div
      class="container"
    >
      <div
        class="columns"
      >
        <div
          class="column is-half"
        >
          <div
            class="content is-medium"
          >
            <h6>
              My Language Learner
            </h6>
            <hr />
            <p>
              Designed, Coded, and Created by Ethan Degenhardt
            </p>
            <p>
              ethandeg1996@gmail.com
            </p>
            <p>
              (636) 373-3435
            </p>
          </div>
        </div>
        <div
          class="column is-half"
        >
          <div
            class="content is-medium"
          >
            <h6>
              Creation
            </h6>
            <hr />
            <p
              class=""
            >
              This app was made with the following technology stack:
            </p>
            <ul
              class="content is-small"
            >
              <li>
                NodeJS
              </li>
              <li>
                Express
              </li>
              <li>
                React
              </li>
              <li>
                Redux
              </li>
              <li>
                Bulma CSS
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </footer>
</DocumentFragment>
`;
